android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -127,6 +132,20 @@ public void setup(ReadableMap options) {
         VoiceConnectionService.setInitialized(true);
         this._settings = options;
 
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            if(isSelfManaged()) {
+                Log.d(TAG, "API Version supports self managed, and is enabled in setup");
+            }
+            else {
+                Log.d(TAG, "API Version supports self managed, but it is not enabled in setup");
+            }
+        }
```

Comment:
This just provides us some logging so we can confirm we're running in self managed mode. Not strictly needed, but convenient.